internal/runtime/delete.go

```diff
@@ -51,7 +51,7 @@ func forceDelete(logger *slog.Logger, containerID string) {
 		}
 	}
 
-	if err := remoteproc.RemoveFirmware(state.Annotations[oci.StateFirmware]); err != nil {
+	if err := remoteproc.RemoveFirmware(state.Annotations[oci.StateStoredFirmwareName]); err != nil {
```

Comment:
It's possible that `state.Annotations[oci.StateStoredFirmwareName]` is empty, in which case this code will error. I don't think it should error in that case.

```bash
./remoteproc-runtime create ./my-fancy-bundle a-container-id
./remoteproc-runtime delete a-container-id
```